release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -558,6 +558,8 @@ For more information, see _Configuring Capacity Reservation by using machine set
 [id="ocp-4-17-nodes_{context}"]
 === Nodes
 
```

Comment:
This should have a heading. Example:
[id="ocp-4-17-enhancements-nodes-crio-wipe_{context}"]
==== New CRI-O command behavior

Reply:
Thanks! I've added a heading.